pkg/scheduler/api/resource_info.go

```diff
@@ -429,6 +429,42 @@ func (r *Resource) LessEqual(rr *Resource, defaultValue DimensionDefaultValue) b
 	return true
 }
 
+// LessEqualWithDimension only compare the resource items in req param
+// @param req define the resource item to be compared
+// if req is nil, equals r.LessEqual(rr, zero)
+func (r *Resource) LessEqualWithDimension(rr *Resource, req *Resource) bool {
```

Comment:
How about just remove the dimensions not in `req` of `r`, then compare just like before?

Reply:
also ok.

Reply:
I think this way is more readable.